community-store.html

```diff
@@ -0,0 +1,262 @@
+            <a href="#">
+              <img class="media-object"
+                ix-src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-header-imgs-nicaragua.jpg?w=108&h=108&fit=crop&format=auto,compress"
+                alt="">
+            </a>
+          </div>
+          <div class="media-body">
+            <div class="group-cell-text">
+              <p class="font-family-condensed-extra text-uppercase">
+                Onsite Groups
+              </p>
+              <span class="font-size-xs">
+                Some quick example text to build on the card title and make up the bulk of the card's content.
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="group-cell  col-sm-6">
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object"
+                ix-src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-header-imgs-nicaragua.jpg?w=108&h=108&fit=crop&format=auto,compress"
+                alt="">
+            </a>
+          </div>
+          <div class="media-body">
+            <div class="group-cell-text">
+              <p class="font-family-condensed-extra text-uppercase">
+                At home groups
+              </p>
+              <span class="font-size-xs">
+                Some quick example text to build on the card title and make up the bulk of the card's content.
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="group-cell  col-sm-6">
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object"
+                ix-src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-header-imgs-nicaragua.jpg?w=108&h=108&fit=crop&format=auto,compress"
+                alt="">
+            </a>
+          </div>
+          <div class="media-body">
+            <div class="group-cell-text">
+              <p class="font-family-condensed-extra text-uppercase">
+                Chaser groups
+              </p>
+              <span class="font-size-xs">
+                Some quick example text to build on the card title and make up the bulk of the card's content.
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <a class="btn btn-lg btn-secondary" href="#">View groups</a>
+  </div>
+</div>
+
+<div id="trips-container" class="jumbotron"
+  style="background-image: url('https://crds-cms-uploads.imgix.net/Uploads/community-store/community-store-trips-bg%403x.jpg?auto=format,compress&fit=crop');">
+  <img class="img-responsive imgix-fluid push-bottom mobile-flush-bottom" alt="" title=""
+    src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-logo-white.png">
+
+  <h1 class="font-family-condensed-extra text-uppercase font-size-h1 flush-top">you were born for adventure</h1>
```

Comment:
Can you convert this element to an `<h2>`? There should only be one `<h1>` element per page.